includes/widgets/class-wc-widget-layered-nav.php

```diff
@@ -423,7 +423,7 @@ protected function layered_nav_list( $terms, $taxonomy, $query_type ) {
 				continue;
 			}
 
-			$filter_name    = 'filter_' . sanitize_title( str_replace( 'pa_', '', $taxonomy ) );
+			$filter_name    = 'filter_' . str_replace( 'pa_', '', $taxonomy );
```

Comment:
My only concern here is by removing the sanitize_title call you introduce the probability of spaces in the filter name, probably not an issue since browsers auto convert these for URLs, just want to be careful we do not change something that can cause a bug. If you are happy with the change then you can ignore my comment.